### DISCUSSION TITLE
Discord Server for Wails?

### DISCUSSION THREAD

**Discussion by KiddoV** [wailsapp/wails#1897] [Ideas]
I think we should have a Discord server for Wails.
A place where we can chat, hangout, get help better (immediately), give more ideas, track GitHub issues and even ask quick or "stupid" questions without opening discussions or issues, blah.. blah... 😅

**Comment by leaanthony** [comment]
We have a slack channel... Details at the bottom of the website 👍

**Comment by KiddoV** [comment]
I see we now have the Discord server. Great!

**Reply by leaanthony** [reply]
When you joining?

**Reply by KiddoV** [reply]
I did. Just after I updated this post :))
